### ISSUE TITLE
Telewest GB server redundant and not in use - comes up as fastest?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#126]
```
What steps will reproduce the problem?
1. run test
2. telewest server in list
3. set telewest as DNS and reboot, pages wont load

What is the expected output? What do you see instead?
error from Virgin with new IP's to try

What version of the product are you using? On what operating system?
version 1.2 / windows7

Please provide any additional information below.


```

Original issue reported on code.google.com by `chunkyhu...@gmail.com` on 29 May 2010 at 5:29

**Comment by GoogleCodeExporter** [comment]
```
Strange. There are two Telewest IP's in the list:

62.31.176.39
193.38.113.3

Do you know which one you were using?

```

Original comment by `helixblue` on 29 May 2010 at 7:51

**Comment by GoogleCodeExporter** [comment]
```
Looking at the error messages I got back from a recent test, was it 
194.117.134.19 ?

194.117.134.19  BlueYonder Tel GB  32  ms | www.facebook.com appears incorrect: 
92.238.96.18, 
static.ak.fbcdn.net appears incorrect: 92.238.96.18, NXDOMAIN Hijacking, 
a.root-servers.net appears 
incorrect: 92.238.96.18, google.com appears incorrect: 92.238.96.18, 
www.google.com is hijacked: 
92.238.96.18, windowsupdate.microsoft.com is hijacked: 92.238.96.18, 
www.paypal.com is hijacked: 
92.238.96.18, twitter.com appears incorrect: 92.238.96.18, 
safebrowsing.clients.google.com appears 
incorrect: 92.238.96.18, www.google-analytics.com appears incorrect: 
92.238.96.18

We don't mark hijacking as a fatal issue for DNS servers, but perhaps we should 
if they fail so many of them.


```

Original comment by `helixblue` on 29 May 2010 at 7:55

**Comment by GoogleCodeExporter** [comment]
```
sorry it was the BlueYonder GB (which is/used to be telewest, now Virgin) 
194.117.134.19
- I use the IP it suggested above and reboot then browser message from Virgin 
saying
these ones are no longer in use and to use new ones instead... when I use DNS
Benchmark program (not namebench) it also says the DNS im using is redundant 
and that
NTT is fastest with 29ms avg - then when I set it to NTT and use your namebench 
app
it says BlueYonder is 169% faster than NTT with 38ms avg ...?
im confused as how namebench can communicate with 194.117.134.19 when its 
redundant?
```

Original comment by `chunkyhu...@gmail.com` on 29 May 2010 at 8:01

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r910.
```

Original comment by `thomas.r...@gmail.com` on 29 May 2010 at 8:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks! I've removed the bad DNS server from the listing, and updated the code 
so that if a server hijacks too 
many hosts (7+), that it is disabled.

The reason it was showing up as such a fast DNS server is because it was lying 
about every request. We've never 
seen this behaviour before, so it wasn't flagging the DNS server as completely 
unable, and instead issued 
warnings.

Both changes will appear in the 1.3-RC1 that will be uploaded later tonight.
```

Original comment by `helixblue` on 29 May 2010 at 8:10
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3-RC4

```

Original comment by `helixblue` on 30 May 2010 at 11:18
- Changed state: **Fixed**
